autogen.sh

```diff
@@ -131,7 +131,7 @@ do
   shift
 done
 
-conf_flags="--enable-maintainer-mode --enable-compile-warnings" #--enable-iso-c
+conf_flags="--enable-maintainer-mode" #--enable-iso-c
```

Comment:
> Running ./configure --enable-maintainer-mode --enable-compile-warnings --without-x11 --enable-asan ...
configure: WARNING: unrecognized options: --enable-compile-warnings

Should I also remove `#-enable-iso-c`